src/main/java/seedu/address/logic/commands/StatisticsCommand.java

```diff
@@ -26,7 +26,7 @@ public class StatisticsCommand extends Command {
     public static final String MESSAGE_DISPLAY_HIGH_PRIORITY = "Number Of HIGH Priority People: %s";
     public static final String MESSAGE_DISPLAY_MEDIUM_PRIORITY = "Number Of MEDIUM Priority People: %s";
     public static final String MESSAGE_DISPLAY_LOW_PRIORITY = "Number Of LOW Priority People: %s";
-    public static final String MESSAGE_DISPLAY_LOW_INCOME = "Number Of Income < 2500 People: %s";
+    public static final String MESSAGE_DISPLAY_LOW_INCOME = "Number Of People Eligible For ComCare: %s";
```

Comment:
is just one of the most common financial assistance scheme from what I gathered online

Reply:
I think its better to not be too specific to any financial aid schemes because there really isn't like a concrete line where we can definitely tell who is qualified for it and not

Reply:
Noted ive changed it to just income range.